script/Ciel/Day 9/09_01_CIEL09_6.txt

```diff
@@ -430,8 +430,9 @@ Look at all that superfluous flab; what are you, a dying hippo?
 [sha:88a9c04f69c433aa68fe9aae07d4571ff61bc6a7]{
 -- Page 568, Offset 23595.
 -- “起きて”“殺す”事だけが貴女の美点でしょうに。人間に助けを求めるなんて、そんな余分はらしくありません。
+// Second sentence needs work
 Superfluous is exactly what I'd label your uncharacteristic behavior of seeking help from a human.
- And here I thought 'waking up' and 'killing' were your only two defining traits.
+ What made you so charming was always the fact that all you did was 'wake up' and 'kill'.
```

Comment:
with hindsight, both this version and your suggestion during gpr are good
> Your charm was always the fact that all you did is 'wake up' and 'kill'.

Reply:
feels like a weird tense mismatch with 'is'